Tanasee/Tanasee1.xml

```diff
@@ -33,16 +36,96 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Kǝbrān 1</idno>
                         </altIdentifier>
                     </msIdentifier>
```

Comment:
EMIP 3496 should possibly be added as another altIdentifier

Reply:
Yes, thanks!